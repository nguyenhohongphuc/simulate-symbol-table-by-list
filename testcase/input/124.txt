INSERT x number
BEGIN
INSERT y string
BEGIN
INSERT z number
PRINT
END
RPRINT
END
LOOKUP x