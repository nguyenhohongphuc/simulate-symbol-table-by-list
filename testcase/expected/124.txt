success
success
success
x//0 y//1 z//2
y//1 x//0
0